app/controllers/games_controller.rb

```diff
@@ -1,4 +1,7 @@
 class GamesController < ApplicationController
+  before_action :authenticate_user!, only: [
```

Comment:
I would prefer using `except` instead of `only` in this case:

```ruby
before_action :authenticate_user!, except: [:index, :show]
```

This way you're **whitelisting** actions (only index and show don't require user authentication) whereas the default is to assume everything else is protected. This ensures that if you ever add a new action it will be authenticated by default.